x-pack/plugins/security_solution/public/management/pages/event_filters/view/event_filters_list_page.test.tsx

```diff
@@ -177,6 +184,28 @@ describe('When on the Event Filters List Page', () => {
     });
   });
 
+  describe('And policies select is dispatched', () => {
+    it('should apply policy filter', async () => {
+      const policies = await sendGetEndpointSpecificPackagePoliciesMock();
+      (sendGetEndpointSpecificPackagePolicies as jest.Mock).mockResolvedValue(policies);
+
+      renderResult = render();
+      await waitFor(() => {
+        expect(sendGetEndpointSpecificPackagePolicies).toHaveBeenCalled();
+      });
+
+      const firstPolicy = policies.items[0];
+
+      fireEvent.click(renderResult.getByTestId('policiesSelectorButton'));
+      await act(async () => {
+        fireEvent.click(
```

Comment:
nit: use `userEvent`

Reply:
I used the act + fireEvent to have the waitFor inside the act. Without the act and using userEvent i had a jest timeout for the waitFor

Reply:
that's a valid use case. you can alternatively use `waitFor` too https://testing-library.com/docs/dom-testing-library/api-async/#waitfor

I don't mind if you leave it like this though.

Reply:
changes in https://github.com/elastic/kibana/pull/121305/commits/d5ac895d5ad8b5969df21e3387f9345ce31ea65e